Fix top margin behaviour for no logo pages

CTalvio opened the pull request:
This uses a negative margin .detaiLogo instead of zeroing the margin on .detalRibbon, to align the logo with the ribbon.

This leaves .detailRibbon without changes, meaning it can keep it's own code, so it doesn't go weird for items with no logo.

CTalvio commented:
![image](https://user-images.githubusercontent.com/4365015/127756493-3485b22d-983a-4a32-b902-e8975df57108.png)
![image](https://user-images.githubusercontent.com/4365015/127756555-57fa728a-b216-40e8-a0e4-b34cabfd5fe9.png)
With and without, before.

![image](https://user-images.githubusercontent.com/4365015/127756540-d2cf3736-b9f5-4a6a-90db-ad68c961086d.png)
![image](https://user-images.githubusercontent.com/4365015/127756505-2d3b7467-eff9-42eb-ac23-d48c08a610c1.png)
With and without, with edit.

prayag17 commented:
I am not able to replicate the issue and I don't see anything wrong right now with logos.
Try using:

```css
 @import url("https://cdn.jsdelivr.net/gh/prayag17/JellySkin@latest/addons/Logo.css");
```

CTalvio commented:
I'm not sure you understood the issue. This is not a critical problem, but the way your logos work right now, on shows that do not have a logo (in my example pictures I deleted it) the page alignment goes way higher than on pages with logos.

I'm talking about the empty space above the Logo/Title. It disappears in your theme if there is no logo, and the show title ends up right at the top of the screen. This looks bad.

This is because enabling the logo code modifies the margins on .detailRibbon, even when there is no logo for a show. I changed it so that it doesn't.

PS: You have a type in your readme, the first import URL, it says "ur" instead of "url"

danieladov commented:
I'm having the same problem

prayag17 commented:
The issue should be fixed, use the import code in readme to  see the changes
